<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitTestGenerator</class>
 <widget class="QMainWindow" name="UnitTestGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UnitTestGenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_populations">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>1201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Популяция 1</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1181</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Популяция 2</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1181</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Популяция 3</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1181</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Популяция 4</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1181</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
